### PR TITLE
Check document verification results

### PR DESCRIPTION
**Pull request by saalistaja**
- Process all results
- Decide about final identity verification status

### CODE REVIEW

File: enrollment-server/src/main/java/com/wultra/app/enrollmentserver/impl/service/verification/VerificationProcessingService.java [hunk 1]
```diff
@@ -0,0 +1,112 @@
+/*
+ * PowerAuth Command-line utility
```
Comment (romanstrobl) [inline enrollment-server/src/main/java/com/wultra/app/enrollmentserver/impl/service/verification/VerificationProcessingService.java:2]: Invalid copyright.

File: enrollment-server/src/main/java/com/wultra/app/enrollmentserver/impl/service/IdentityVerificationService.java [hunk 2]
```diff
@@ -239,6 +288,10 @@ public DocumentStatusResponse checkIdentityVerificationStatus(DocumentStatusRequ
         return response;
     }
 
+    public void checkVerificationResult(OwnerId ownerId, IdentityVerificationEntity idVerification) {
+        // TODO get all
```
Comment (romanstrobl) [inline enrollment-server/src/main/java/com/wultra/app/enrollmentserver/impl/service/IdentityVerificationService.java:292]: Is this TODO relevant?

**Review by romanstrobl — APPROVED** [review]
Nice progress. Let's fix the copyright and check one TODO.

File: enrollment-server/src/main/java/com/wultra/app/enrollmentserver/impl/service/verification/VerificationProcessingService.java (hunk 1 above)
Comment (saalistaja) [reply enrollment-server/src/main/java/com/wultra/app/enrollmentserver/impl/service/verification/VerificationProcessingService.java]: aah, fixed

File: enrollment-server/src/main/java/com/wultra/app/enrollmentserver/impl/service/IdentityVerificationService.java (hunk 2 above)
Comment (saalistaja) [reply enrollment-server/src/main/java/com/wultra/app/enrollmentserver/impl/service/IdentityVerificationService.java]: nope, removed

**Pull request merged**